os/mounting-storage.md

```diff
@@ -35,11 +37,13 @@ coreos:
         Description=Formats the ephemeral drive
         After=dev-xvdb.device
         Requires=dev-xvdb.device
+        ConditionPathExists=!/var/lib/format-xvdb
```

Comment:
I don't think we should support that use case. Adding storage after the fact is a very manual operation which goes against nearly everything we produce. We should instead document a fix for that user's root cause.